nuget/NUnit3TestAdapter.nuspec

```diff
@@ -24,6 +24,19 @@
     <tags>test visualstudio testadapter nunit nunit3 dotnet</tags>
 
     <developmentDependency>false</developmentDependency>
+
+    <dependencies>
+      <group targetFramework="net462">
+        <dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="1.0.1" />
```

Comment:
We recommend having explicit deps to the bridge rather than repackaging the dlls/props/targets... to ease and standardize versioning/update conflicts.